src/Symfony/Component/Validator/ConstraintValidator.php

```diff
@@ -105,6 +111,37 @@ protected function formatValue($value, $format = 0)
             return $value->format('Y-m-d H:i:s');
         }
 
+        if (($format & self::PRETTY_DATE_INTERVAL) && $value instanceof \DateInterval) {
+            $formattedValueParts = [];
+            foreach ([
```

Comment:
What about putting it in a private constant ?

Reply:
What would it bring in your opinion?

For me it's fine like this because it's not something I want to expose. The value is easily understandable, immutable, not reused, and will very likely never be modified in the future.